samples/AutomaticTraceIdInjection/Log4NetExample/Log4NetExample.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Datadog.Trace" Version="1.26.1" />
+    <PackageReference Include="Datadog.Trace" Version="0.0.1" />
```

Comment:
Probably not a blocker for this PR, but none of these "sample" projects will build with a NuGet reference to `Datadog.Trace` version 0.0.1 (such a packages does not exist).